x-pack/plugins/siem/public/components/load_more_table/index.mock.tsx

```diff
@@ -46,7 +46,12 @@ export const mockData = {
   },
 };
 
-export const getHostsColumns = (): Array<Columns<string>> => [
+export const getHostsColumns = (): [
+  Columns<string>,
+  Columns<string>,
+  Columns<string>,
+  Columns<string>
+] => [
```

Comment:
Since these are all the same type, you could use `Array<Columns<string>>` if you wanted since subsequent columns will be assigned type `T` as per our new `BasicTableProps` typings:

``` typescript
BasicTableProps<T, U = T, V = T, W = T, X = T, Y = T, Z = T, AA = T, AB = T>
```

That said, being explicit like this is nice for maintenance + easier to grok, so worth keeping as it. Either way, nice to now be able to have such clarity with these types! :)

Reply:
The issue was that, I had to remove the `Array<Columns<string>>` as that was interfering with the `T, U, V, etc..` types. Sad I know. Something I can probably work out later. The great news is that I was able to remove the `any` from DateTime and now we are `any` free.

Reply:
That's _fantastic_. I was curious if we could now remove the DateScalar `any` with these latest changes, so this a great improvement!